src/psyclone/tests/test_files/gocean1p0/test00.1_invoke_kernel_using_const_scalar.f90

```diff
@@ -0,0 +1,24 @@
+!-------------------------------------------------------------------------------
+! (c) The copyright relating to this work is owned jointly by the Crown,
+! Met Office and NERC 2015.
+! However, it has been created with the help of the GungHo Consortium,
+! whose members are identified at https://puma.nerc.ac.uk/trac/GungHo/wiki
+!-------------------------------------------------------------------------------
+! Author A. Porter STFC Daresbury Lab
```

Comment:
Please make yourself the author and remove the bit about funding from the GOcean project.